src/core/server/ui_settings/saved_objects/ui_settings.ts

```diff
@@ -9,6 +9,14 @@
 import { SavedObjectsType } from '../../saved_objects';
 import { migrations } from './migrations';
 
+/**
+ * The `config` object type contains many attributes that are defined by consumers.
+ */
+export interface ConfigAttributes {
+  buildNum: number;
```

Comment:
NIT: should we add `isDefaultIndexMigrated` here too?

Reply:
I don't think so, this was to address Luke's feedback in https://github.com/elastic/kibana/pull/133339#discussion_r889382279, we have a separate `UpgradeableConfigAttributes` type that extends `ConfigAttributes` and specifies other fields such as `isDefaultIndexMigrated`.